tutorials/models/1_gnn/1_gcn.py

```diff
@@ -137,8 +155,9 @@ def load_cora_data():
     if epoch >=3:
         dur.append(time.time() - t0)
     
-    print("Epoch {:05d} | Loss {:.4f} | Time(s) {:.4f}".format(
-            epoch, loss.item(), np.mean(dur)))
+    acc = evaluate(net, g, features, labels, test_mask)
+    print("Epoch {:05d} | Loss {:.4f} | Acc {:.4f} | Time(s) {:.4f}".format(

```

Comment:
`print("Epoch {:05d} | Loss {:.4f} | Acc {:.4f} | Time(s) {:.4f}" `
to
`print("Epoch {:05d} | Train Loss {:.4f} | Test Acc {:.4f} | Time(s) {:.4f}" `